style/desktop.css

```diff
@@ -240,12 +241,330 @@
     /* width: 400px; */
 
```

Comment:
Kindly I suggest you remove all commented code and white lines in your code base to follow best practices, clean code, and be more professional. 😄 

[OPTIONAL] I suggest you use `grid` instead of `flex` to align the cards in the project section.

Please make sure you replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### The Figma design
>
><img width="644" alt="Screenshot 2023-05-04 at 5 06 14 AM" src="https://user-images.githubusercontent.com/91237525/236162783-917a3f3a-8053-4bd0-a2c7-c9e81255732e.png">
>
>### Your design
>
><img width="1393" alt="Screenshot 2023-05-04 at 5 20 09 AM" src="https://user-images.githubusercontent.com/91237525/236163117-04c61601-cc08-4b86-b1c1-2d53edff8279.png">